cmd/internal/cli/key_export.go

```diff
@@ -14,14 +14,14 @@ import (
 	"github.com/sylabs/singularity/pkg/sypgp"
 )
 
-var (
-	secretExport bool
-)
+var secretExport bool
+var armor bool
 
 func init() {
 	KeyExportCmd.Flags().SetInterspersed(false)
 
 	KeyExportCmd.Flags().BoolVarP(&secretExport, "secret", "s", false, "export a secret key")
+	KeyExportCmd.Flags().BoolVarP(&armor, "armor", "a", false, "key armored format")
```

Comment:
maybe `use ascii armored format`?
```suggestion
	KeyExportCmd.Flags().BoolVarP(&armor, "armor", "a", false, "use ascii armored format")
```